packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -630,6 +630,10 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
         role="option"
         aria-selected={isItemSelected ? 'true' : 'false'}
         checked={isItemSelected}
+        // Since the Checkbox is triggered via a click on either the checkbox itself or its label, we need to make the label span the
+        // entire width of the Dropdown so users can select and unselect the checkbox as long as they are clicking in the area adjacent
+        // to it.
+        styles={{ label: { width: '100%' } }}
```

Comment:
any way here to provide this in a styles file and not inline?

Reply:
I don't think that this is the right fix, it still won't expand to fill the entire container height wise. So the clickbox still isn't the whole highlightable space.

Reply:
@dzearing I'll move the styles to a styles file.

@joschect You're right, I missed that part, great catch.